contracts/amm/spread/CalculateTimeWeightedNotionalLibs.sol

```diff
@@ -115,23 +117,48 @@ library CalculateTimeWeightedNotionalLibs {
 
         for (uint256 i; i != length; ) {
             timeWeightedNotional = SpreadStorageLibs.getTimeWeightedNotional(timeWeightedNotionalStorageIds[i]);
```

Comment:
I will change name SpreadStorageLibs.getTimeWeightedNotional( to SpreadStorageLibs.getTimeWeightedNotionalForAssetAndTenor(

Reply:
The same with saveTimeWeightedNotional change to saveTimeWeightedNotionalForAssetAndTenor